pages/osx/colima.md

```diff
@@ -0,0 +1,36 @@
+# colima
+
+> Container runtimes for macOS and Linux with minimal setup.
+> More information: <https://github.com/abiosoft/colima>.
+
+- Start Colima:
+
+`colima start`
+
+- Use a config file:
+
+`colima start --edit`
+
+- Start and setup Containerd. Install nerdctl to use Containerd via nerdctl:
+
+`colima start --runtime containerd`
+
+- Start with Kubernetes. `kubectl` is required:
+
+`colima start --kubernetes`
+
+- Customize CPU, memory, disk:
```

Comment:
This is unclear. Is it CPU threads, cores? What is the unit of the memory option, MB, MiB, GB? What's the unit of the disk option?